src/lovelace/settings.py

```diff
@@ -76,8 +80,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'projectlovelace',
+        'USER': 'admin',
+        'PASSWORD': 'foobar',
```

Comment:
The database password, Django secret key, and debug option are settings we will update on deployment (whether it's automated or not).